### ISSUE TITLE
Release Pythonnet 2.1

### ISSUE THREAD

**Issue by vmuriart** [pythonnet/pythonnet#183]
There's been alot of changes & improvements since `2.0` release, even `2.1dev1`. (adding python 3.x upto 3.5). Perhaps its time to release `2.1` and make it available in PyPi.

**Comment by vmuriart** [comment]
@tonyroberts Forgot to tag you in the question. 
We mentioned it on the Python 3.5 thread that it seemed about the right time to release, anything in particular holding this back?

**Comment by tonyroberts** [comment]
There are some outstanding problems with type conversion that need to be addressed first. Unfortunately these problems were introduced in an attempt to fix some other problems, but appear to be causing more problems (see PR #176 for the latest attempt to fix). There are no other blocking problems I'm aware of.

**Comment by stonebig** [comment]
no progress on this ?

**Comment by denfromufa** [comment]
I'm not sure what @tonyroberts is waiting on?

**Comment by vmuriart** [comment]
There were a few open tickets he wanted to close first hat I think broke
previous features

**Comment by stonebig** [comment]
someone told me that the wheel created during test where somehow downloadable, but I don't find where they could be

**Comment by denfromufa** [comment]
@vmuriart pull request #176 that was blocking release is merged - not sure which tickets you mean? But maybe @tonyroberts is waiting for fix on #190

**Comment by denfromufa** [comment]
@stonebig here is the link to sample wheels on appveyor:

https://ci.appveyor.com/project/TonyRoberts/pythonnet-480xs/build/1.0.105/job/8yo6slgn41ffo8hp/artifacts

if you navigate to jobs and select python version, then builds are located under artifacts.

**Comment by stonebig** [comment]
thanks! Unfortunately it's still failing hard on wordpad demo.

**Comment by denfromufa** [comment]
@stonebig what is the error you get when you launch wordpad demo from
command line? which version (2.7..3.5, 32-64bit) did you try?

**Comment by stonebig** [comment]
using pythonnet-2.1.0.dev1-cp34-cp34m-win_amd64.whl from you pinpointed Merge pull request #176 from denfromufa/patch-8:
- launchiing wordpad.py from IDLEX => ok, the windows appear,
- "Help -> About" gives "TypeError: No Method matches given argument". detail error below.. cursiously it speaks about Win32
````
Consultez la fin de ce message pour plus de détails sur l'appel du débogage
juste-à-temps (JIT) à la place de cette boîte de dialogue.

************** Texte de l'exception **************
Python.Runtime.PythonException: TypeError : No method matches given arguments
   à Python.Runtime.Dispatcher.Dispatch(ArrayList args)
   à System.Windows.Forms.MenuItem.OnClick(EventArgs e)
   à System.Windows.Forms.MenuItem.MenuItemData.Execute()
   à System.Windows.Forms.Command.Invoke()
   à System.Windows.Forms.Control.WmCommand(Message& m)
   à System.Windows.Forms.Control.WndProc(Message& m)
   à System.Windows.Forms.Form.WndProc(Message& m)
   à System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Assemblys chargés **************
mscorlib
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
clrmodule
    Version de l'assembly : 1.0.0.0
    Version Win32 : 1.0.0.0
    CodeBase : file:///D:/WinPython/basedir34/build/winpython-64bit-3.4.x.2/python-3.4.4.amd64/lib/site-packages/clr.pyd
----------------------------------------
Python.Runtime
    Version de l'assembly : 4.0.0.1
    Version Win32 : 2.0.0.2
    CodeBase : file:///D:/WinPython/basedir34/build/winpython-64bit-3.4.x.2/python-3.4.4.amd64/lib/site-packages/Python.Runtime.dll
----------------------------------------
System.Core
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
__CodeGenerator_Assembly
    Version de l'assembly : 0.0.0.0
    Version Win32 : 2.0.0.2
    CodeBase : file:///D:/WinPython/basedir34/build/winpython-64bit-3.4.x.2/python-3.4.4.amd64/lib/site-packages/Python.Runtime.dll
----------------------------------------
e__NativeCall_Assembly
    Version de l'assembly : 0.0.0.0
    Version Win32 : 2.0.0.2
    CodeBase : file:///D:/WinPython/basedir34/build/winpython-64bit-3.4.x.2/python-3.4.4.amd64/lib/site-packages/Python.Runtime.dll
----------------------------------------
System.Security
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1073.0 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Security/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Security.dll
----------------------------------------
System.Xml
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1067.0 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Configuration
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Data.SqlXml
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1067.0 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Data.SqlXml/v4.0_4.0.0.0__b77a5c561934e089/System.Data.SqlXml.dll
----------------------------------------
System.Windows
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Windows.dll
----------------------------------------
System.Windows.Forms
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System.Drawing
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1068.2 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
Accessibility
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
System.Windows.Forms.resources
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms.resources/v4.0_4.0.0.0_fr_b77a5c561934e089/System.Windows.Forms.resources.dll
----------------------------------------
mscorlib.resources
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/mscorlib.resources/v4.0_4.0.0.0_fr_b77a5c561934e089/mscorlib.resources.dll
----------------------------------------

************** Débogage JIT **************
Pour activer le débogage juste-à-temps (JIT), le fichier de configuration pour cette
application ou cet ordinateur (machine.config) doit avoir la valeur
jitDebugging définie dans la section system.windows.forms.
L'application doit également être compilée avec le débogage
activé.

Par exemple :

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

Lorsque le débogage juste-à-temps est activé, les exceptions non gérées
seront envoyées au débogueur JIT inscrit sur l'ordinateur
plutôt que d'être gérées par cette boîte de dialogue.
````

- "File -> Save" stalls without showing anything

**Comment by stonebig** [comment]
same experience with Python3.4 32 bit and pythonnet-2.1.0.dev1-cp34-cp34m-win32.whl gives:
````
consultez la fin de ce message pour plus de détails sur l'appel du débogage
juste-à-temps (JIT) à la place de cette boîte de dialogue.

************** Texte de l'exception **************
Python.Runtime.PythonException: TypeError : No method matches given arguments
   à Python.Runtime.Dispatcher.Dispatch(ArrayList args)
   à __System_EventHandlerDispatcher.Invoke(Object , EventArgs )
   à System.Windows.Forms.MenuItem.OnClick(EventArgs e)
   à System.Windows.Forms.MenuItem.MenuItemData.Execute()
   à System.Windows.Forms.Command.Invoke()
   à System.Windows.Forms.Command.DispatchID(Int32 id)
   à System.Windows.Forms.Control.WmCommand(Message& m)
   à System.Windows.Forms.Control.WndProc(Message& m)
   à System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   à System.Windows.Forms.Form.WndProc(Message& m)
   à System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   à System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   à System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Assemblys chargés **************
mscorlib
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll
----------------------------------------
clrmodule
    Version de l'assembly : 1.0.0.0
    Version Win32 : 1.0.0.0
    CodeBase : file:///D:/WinPython/basedir34/build/winpython-32bit-3.4.x.2/python-3.4.4/lib/site-packages/clr.pyd
----------------------------------------
Python.Runtime
    Version de l'assembly : 4.0.0.1
    Version Win32 : 2.0.0.2
    CodeBase : file:///D:/WinPython/basedir34/build/winpython-32bit-3.4.x.2/python-3.4.4/lib/site-packages/Python.Runtime.dll
----------------------------------------
System.Core
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
__CodeGenerator_Assembly
    Version de l'assembly : 0.0.0.0
    Version Win32 : 2.0.0.2
    CodeBase : file:///D:/WinPython/basedir34/build/winpython-32bit-3.4.x.2/python-3.4.4/lib/site-packages/Python.Runtime.dll
----------------------------------------
e__NativeCall_Assembly
    Version de l'assembly : 0.0.0.0
    Version Win32 : 2.0.0.2
    CodeBase : file:///D:/WinPython/basedir34/build/winpython-32bit-3.4.x.2/python-3.4.4/lib/site-packages/Python.Runtime.dll
----------------------------------------
System.Security
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1073.0 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Security/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Security.dll
----------------------------------------
System.Xml
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1067.0 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Configuration
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Data.SqlXml
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1067.0 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Data.SqlXml/v4.0_4.0.0.0__b77a5c561934e089/System.Data.SqlXml.dll
----------------------------------------
System.Windows
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Windows.dll
----------------------------------------
System.Windows.Forms
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System.Drawing
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1068.2 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
Accessibility
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
System.Windows.Forms.resources
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms.resources/v4.0_4.0.0.0_fr_b77a5c561934e089/System.Windows.Forms.resources.dll
----------------------------------------
mscorlib.resources
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/mscorlib.resources/v4.0_4.0.0.0_fr_b77a5c561934e089/mscorlib.resources.dll
----------------------------------------

************** Débogage JIT **************
Pour activer le débogage juste-à-temps (JIT), le fichier de configuration pour cette
application ou cet ordinateur (machine.config) doit avoir la valeur
jitDebugging définie dans la section system.windows.forms.
L'application doit également être compilée avec le débogage
activé.

Par exemple :

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

Lorsque le débogage juste-à-temps est activé, les exceptions non gérées
seront envoyées au débogueur JIT inscrit sur l'ordinateur
plutôt que d'être gérées par cette boîte de dialogue.
````

**Comment by stonebig** [comment]
blows up also with 2.7:
````
Consultez la fin de ce message pour plus de détails sur l'appel du débogage
juste-à-temps (JIT) à la place de cette boîte de dialogue.

************** Texte de l'exception **************
Python.Runtime.PythonException: TypeError : No method matches given arguments
   à Python.Runtime.Dispatcher.Dispatch(ArrayList args)
   à __System_EventHandlerDispatcher.Invoke(Object , EventArgs )
   à System.Windows.Forms.MenuItem.OnClick(EventArgs e)
   à System.Windows.Forms.MenuItem.MenuItemData.Execute()
   à System.Windows.Forms.Command.Invoke()
   à System.Windows.Forms.Command.DispatchID(Int32 id)
   à System.Windows.Forms.Control.WmCommand(Message& m)
   à System.Windows.Forms.Control.WndProc(Message& m)
   à System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   à System.Windows.Forms.Form.WndProc(Message& m)
   à System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   à System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   à System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Assemblys chargés **************
mscorlib
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll
----------------------------------------
clrmodule
    Version de l'assembly : 1.0.0.0
    Version Win32 : 1.0.0.0
    CodeBase : file:///D:/result_tests/winpython-2.7.10.3/python-2.7.10/lib/site-packages/clr.pyd
----------------------------------------
Python.Runtime
    Version de l'assembly : 4.0.0.1
    Version Win32 : 2.0.0.2
    CodeBase : file:///D:/result_tests/winpython-2.7.10.3/python-2.7.10/lib/site-packages/Python.Runtime.dll
----------------------------------------
System.Core
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
__CodeGenerator_Assembly
    Version de l'assembly : 0.0.0.0
    Version Win32 : 2.0.0.2
    CodeBase : file:///D:/result_tests/winpython-2.7.10.3/python-2.7.10/lib/site-packages/Python.Runtime.dll
----------------------------------------
e__NativeCall_Assembly
    Version de l'assembly : 0.0.0.0
    Version Win32 : 2.0.0.2
    CodeBase : file:///D:/result_tests/winpython-2.7.10.3/python-2.7.10/lib/site-packages/Python.Runtime.dll
----------------------------------------
System.Security
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1073.0 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Security/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Security.dll
----------------------------------------
System.Xml
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1067.0 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
System.Configuration
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Data.SqlXml
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1067.0 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Data.SqlXml/v4.0_4.0.0.0__b77a5c561934e089/System.Data.SqlXml.dll
----------------------------------------
System.Windows
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Windows.dll
----------------------------------------
System.Windows.Forms
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System.Drawing
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1068.2 built by: NETFXREL3STAGE
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
Accessibility
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
System.Windows.Forms.resources
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms.resources/v4.0_4.0.0.0_fr_b77a5c561934e089/System.Windows.Forms.resources.dll
----------------------------------------
mscorlib.resources
    Version de l'assembly : 4.0.0.0
    Version Win32 : 4.6.1055.0 built by: NETFXREL2
    CodeBase : file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/mscorlib.resources/v4.0_4.0.0.0_fr_b77a5c561934e089/mscorlib.resources.dll
----------------------------------------

************** Débogage JIT **************
Pour activer le débogage juste-à-temps (JIT), le fichier de configuration pour cette
application ou cet ordinateur (machine.config) doit avoir la valeur
jitDebugging définie dans la section system.windows.forms.
L'application doit également être compilée avec le débogage
activé.

Par exemple :

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

Lorsque le débogage juste-à-temps est activé, les exceptions non gérées
seront envoyées au débogueur JIT inscrit sur l'ordinateur
plutôt que d'être gérées par cette boîte de dialogue.


````

**Comment by vmuriart** [comment]
@stonebig probably best to open a new issue since its off-topic. That being said, has this worked in any of the previous versions?

**Comment by stonebig** [comment]
"about" works now on my PC. thanks !

**Comment by tonyroberts** [comment]
Ok, I think we're good to do that release now. I'll get it ready.

**Comment by vmuriart** [comment]
[Released](https://pypi.python.org/pypi/pythonnet/2.1.0) a few days ago
